python/publish/publisher.py

```diff
@@ -305,6 +307,15 @@ def publish_json(self, data: PublishData):
         # provide a reduced version to Github actions
         self._gha.set_output('json', json.dumps(data.reduced(), ensure_ascii=False))
 
+    def publish_summary(self, data: PublishData):
+        if self._settings.job_summary and  os.getenv("GITHUB_STEP_SUMMARY"):
+            try:
+                with open(os.environ["GITHUB_STEP_SUMMARY"], "a+") as summary:
+                    summary.write(f"\n# {data.title}\n\n")
```

Comment:
In `publish_comment`, the markdown is constructed as: `body = f'## {title}\n{summary}'`. Shall we use that to start with?

Reply:
Good catch, I'll just write simply the summary